squirrel_sensing_node/include/squirrel_sensing_node/node.h

```diff
@@ -10,7 +10,7 @@
 #include "common_defines.h"
 
 //comment below line to disable wrist sensor (in case it is not plugged in)
-#define _FT17_AVAIL
+// #define _FT17_AVAIL
```

Comment:
please do not push this change

Reply:
@mzillich was this part of your fix? I think this is there because our sensor is not operating right?

Reply:
Removed this change